From 91d72cc97138e5349e36bba9329ae29dee814388 Mon Sep 17 00:00:00 2001
From: Liwei Song <liwei.song@windriver.com>
Date: Tue, 2 Jun 2020 09:55:49 +0000
Subject: [PATCH] mkldnn_memory_solver.hpp: include stdint.h to avoid build
 error

fix the following compile error:

inference-engine/src/mkldnn_plugin/mkldnn_memory_solver.hpp:60:9: error: 'int64_t' does not name a type
|    60 |         int64_t size;
|       |         ^~~~~~~

include stdint.h to fix this.

Upstream-Status: Submitted [https://github.com/openvinotoolkit/openvino/pull/729]

Signed-off-by: Liwei Song <liwei.song@windriver.com>
---
 inference-engine/src/mkldnn_plugin/mkldnn_memory_solver.hpp | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/inference-engine/src/mkldnn_plugin/mkldnn_memory_solver.hpp b/inference-engine/src/mkldnn_plugin/mkldnn_memory_solver.hpp
index 2c236a307276..aa52d29b310d 100644
--- a/inference-engine/src/mkldnn_plugin/mkldnn_memory_solver.hpp
+++ b/inference-engine/src/mkldnn_plugin/mkldnn_memory_solver.hpp
@@ -10,6 +10,8 @@
 
 #include "ie_api.h"
 
+#include <stdint.h>
+
 #include <vector>
 #include <map>
 
-- 
2.24.1

